docs/design/admission_control_limit_range.md

```diff
@@ -0,0 +1,122 @@
+# Admission control plugin: LimitRanger
+
+## Background
+
+This document proposes a system for enforcing min/max limits per resource as part of admission control.
```

Comment:
Never mind.  I was thinking this PR included code too.  It is still just outline.  So, ignore above comments.